script/create-redirects.js

```diff
@@ -52,13 +52,16 @@ function createRedirects(options) {
       };
 
       for (const alias of aliases) {
-        let absolutePath = path.join(options.destination, alias);
-        if (!path.extname(absolutePath))
-          absolutePath = path.join(absolutePath, 'index.html');
+        let finalPath;
```

Comment:
Maybe this will break on Jenkins, but the blc expects Metalsmith pages to have a relative path, not an absolute path. Locally, this seems to work for me.